### PR TITLE
Tufgaben7

### PR DESCRIPTION
**Pull request by Reisii**

### CODE REVIEW

File: Exercises/TuNguyenXuan/Tufgaben7.py [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+
+output_list = []
+matrix = np.zeros([23,43])
+ball_position = 0
+paddle_position = 0
+score = 0
+first_frame = True
+
+
+"""root_dir = pathlib.Path(__file__).parent.parent.parent
```
Comment (paulseidel08) [inline Exercises/TuNguyenXuan/Tufgaben7.py:14]: You shouldn't "comment" this "out" in your PR. The idea of the whole pathlib thing is that reviewers don't have to change anything in your code, to run it.

File: Exercises/TuNguyenXuan/Tufgaben7.py [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+with open(command_file) as input_file:
+    data_input_list = list()
+    for line in input_file.readlines():
+        line.strip()
+        data_input_list.append(int(line))
+    data_input_list[0] = 2"""
+
+
+
+with open("breakout_commands.txt", "r") as input_file:
```
Comment (paulseidel08) [inline Exercises/TuNguyenXuan/Tufgaben7.py:26]: In this case, you should "outcomment" this part

File: Exercises/TuNguyenXuan/Tufgaben7.py [hunk 3]
```diff
@@ -0,0 +1,77 @@
+from matplotlib import pyplot as plt
+import pathlib
+import numpy as np
+import computer
```
Comment (paulseidel08) [inline Exercises/TuNguyenXuan/Tufgaben7.py:4]: Nice idea, to put the computer in another file!

**Review by paulseidel08 — APPROVED** [review]
Seems fine to me.
However, u could have used at least some comments in your code.
In the upcoming exercises, please "uncomment" the general importation for the File-Path, as it is on GitHub, so that Reviewers don't have to change your code to run it on their system.

**Comment by fabischw** [comment]
I will be merging this PR if it does not get merged by the PR creator within the next 3 days in an effort to clean up the repo (More information can be found in [Issue #228](https://github.com/dhbw-ma-ppp/ppp-2023/issues/228))

**Pull request merged**